mockgcp/mockcompute/service.go

```diff
@@ -222,8 +223,22 @@ func (s *MockService) NewHTTPMux(ctx context.Context, conn *grpc.ClientConn) (ht
 
 	// Returns slightly non-standard errors
 	mux.RewriteError = func(ctx context.Context, error *httpmux.ErrorResponse) {
-		// Does not return status (at least for 404)
-		error.Status = ""
+		if error.Code == 404 {
+			// Transform the mock error message:
+			// i.e. "address \"projects/${projectId}/global/addresses/computeaddress-${uniqueId}\" not found"
+			// into the realGCP error message:
+			// i.e. "The resource 'projects/${projectId}/global/addresses/computeaddress-${uniqueId}' was not found"
```

Comment:
Can you just return the correct message from the Get method?  That's what we do elsewhere.  It looks like this:

```
if status.Code(err) == codes.NotFound {
  return status.Errorf(codes.NotFound, "The resource '%s' was not found", fqn)
}
```

Or something along those lines, anyway!